agent/functional_tests/testdata/taskdefinitions/gmsa-localfile/task-definition.json

```diff
@@ -0,0 +1,35 @@
+{
+  "containerDefinitions": [
+    {
+      "entryPoint": [
+        "powershell",
+        "-Command"
+      ],
+      "portMappings": [
+        {
+          "hostPort": 8080,
+          "protocol": "tcp",
+          "containerPort": 80
+        }
+      ],
+      "command": [
+        "New-Item -Path C:\\inetpub\\wwwroot\\index.html -ItemType file -Value '<html> <head> <title>Amazon ECS Sample App</title> <style>body {margin-top: 40px; background-color: #333;} </style> </head><body> <div style=color:white;text-align:center> <h1>Amazon ECS Sample App</h1> <h2>Congratulations!</h2> <p>Your application is now running on a container in Amazon ECS.</p>' -Force ; C:\\ServiceMonitor.exe w3svc"
+      ],
+      "cpu": 512,
+      "environment": [],
+      "mountPoints": [],
+      "dockerSecurityOptions": [
```

Comment:
Can you also test S3 and SSM options?

Reply:
Testing S3/SSM would require us to anticipate the ARNs of these objects prior to their creation in the test as they would need to be present in the task definition. 

Also, the SSM ARN would have the account-id which should not be exposed to the outside world.

Reply:
S3/SSM tests will be opened as a different PR.

Reply:
https://github.com/aws/amazon-ecs-agent/pull/2285